src/components/FirmwareUpdateBanner.js

```diff
@@ -0,0 +1,282 @@
+// @flow
+
+import React, { useState, useEffect, useContext } from "react";
+
+import {
+  View,
+  TouchableOpacity,
+  TouchableHighlight,
+  StyleSheet,
+  Platform,
+} from "react-native";
+import Animated, {
+  set,
+  interpolate,
+  useCode,
+  Easing,
+} from "react-native-reanimated";
+import { useClock, timing } from "react-native-redash/lib/module/v1";
+import manager from "@ledgerhq/live-common/lib/manager";
+import { useTheme } from "@react-navigation/native";
+import type {
+  DeviceModelInfo,
+  FirmwareUpdateContext,
+} from "@ledgerhq/live-common/lib/types/manager";
+import { useTranslation } from "react-i18next";
+import { useSelector } from "react-redux";
+import IconExclamation from "../icons/ExclamationCircleFull";
+import ButtonUseTouchable from "../context/ButtonUseTouchable";
+import { lastSeenDeviceSelector } from "../reducers/settings";
+import { BaseButton as Button } from "./Button";
+import IconDownload from "../icons/Download";
+import BottomModal from "./BottomModal";
+import IconClose from "../icons/Close";
+import IconNano from "../icons/NanoS";
+import { rgba } from "../colors";
+import LText from "./LText";
+
+const FirmwareUpdateBanner = () => {
+  const lastSeenDevice: DeviceModelInfo = useSelector(lastSeenDeviceSelector);
+  const [showDrawer, setShowDrawer] = useState<boolean>(false);
+  const [showBanner, setShowBanner] = useState<boolean>(false);
+  const [version, setVersion] = useState<string>("");
+
+  const [anim] = useState(new Animated.Value(0));
+  const clock = useClock();
+  const { colors } = useTheme();
+  const { t } = useTranslation();
+  const useTouchable = useContext(ButtonUseTouchable);
+
+  useEffect(() => {
+    async function getLatestFirmwareForDevice() {
+      const fw: FirmwareUpdateContext = await manager.getLatestFirmwareForDevice(
+        lastSeenDevice?.deviceInfo,
+      );
+
+      setShowBanner(Boolean(fw));
+      setVersion(fw?.final?.version ?? "");
+    }
+
+    getLatestFirmwareForDevice();
+  }, [lastSeenDevice]);
+
+  useCode(
+    () =>
+      set(
+        anim,
+        timing({
+          duration: 500,
+          easing: Easing.inOut(Easing.ease),
+          clock,
+          from: anim,
+          to: new Animated.Value(showBanner ? 0 : 1),
+        }),
+      ),
+    [showBanner],
+  );
+
+  const translateY = interpolate(anim, {
+    inputRange: [0, 1],
+    outputRange: [30, "-100%"],
+  });
+  const opacity = interpolate(anim, {
+    inputRange: [0, 0.4],
+    outputRange: [1, 0],
+  });
+
+  const onPress = () => {
+    setShowDrawer(true);
+  };
+  const onCloseDrawer = () => {
+    setShowDrawer(false);
+  };
+  const onDismissBanner = () => {
+    setShowBanner(false);
+  };
+
+  return (
+    showBanner && (
+      <>
+        <Animated.View
+          style={[
+            styles.banner.root,
+            {
+              opacity,
+              transform: [{ translateY }],
```

Comment:
if its just a simple slide down you could've used [react-native-animatable](https://github.com/oblador/react-native-animatable)
which we already use in the app :sweat_smile: 
If this works fine keep it and happy to see you using reanimated but in case of simple animations keep in mind there are simpler alternatives.

Reply:
Sure ! I just used the same tools that were used by the SnackBar.js, but it felt quite a lot of boilerplate just for this so I'd be happy to switch it if there is something simpler 👍

Reply:
Switched to animatable ! 👍 
Thx for the input !